<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>400</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PnPowerEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水印功率</string>
    </property>
   </widget>
   <widget class="QSlider" name="GSlider">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>119</y>
      <width>19</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="GLabel">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>101</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SniG">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>310</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SniG</string>
    </property>
   </widget>
   <widget class="QSlider" name="ASFminSlider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>119</y>
      <width>19</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ASFminLable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>101</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Asfmin</string>
    </property>
   </widget>
   <widget class="QSlider" name="ASFmaxSlider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>119</y>
      <width>19</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ASFmaxLable">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>101</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Asfmax</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="WMAddCheckBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  添加水印</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="PNaddMUSICqwtPlot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>紧急推送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>标准状态</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SmpRateComboBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>841</x>
      <y>110</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>采样率</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="File_radioButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  读写文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MusicFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>音频文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Music_FilePathEdit">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>40</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>391</y>
      <width>179</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>文件处理进度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_wavfile">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="PnFileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水印文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pn_FilePathEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="NUMComboBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>179</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>台号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NAMEComboBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="OnePnFrameEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RunningLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>OnePnFrameEdit</zorder>
   <zorder>NAMEComboBox</zorder>
   <zorder>Pn_FilePathEdit</zorder>
   <zorder>PnFileButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>StartButton</zorder>
   <zorder>StopButton</zorder>
   <zorder>PnPowerEdit</zorder>
   <zorder>label</zorder>
   <zorder>GSlider</zorder>
   <zorder>GLabel</zorder>
   <zorder>label_SniG</zorder>
   <zorder>ASFminSlider</zorder>
   <zorder>ASFminLable</zorder>
   <zorder>label_3</zorder>
   <zorder>ASFmaxSlider</zorder>
   <zorder>ASFmaxLable</zorder>
   <zorder>label_4</zorder>
   <zorder>WMAddCheckBox</zorder>
   <zorder>PNaddMUSICqwtPlot</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>SmpRateComboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>File_radioButton</zorder>
   <zorder>MusicFileButton</zorder>
   <zorder>Music_FilePathEdit</zorder>
   <zorder>NUMComboBox</zorder>
   <zorder>label_8</zorder>
   <zorder>RunningLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
